code/modules/surgery/bodyparts/head.dm

```diff
@@ -124,6 +124,10 @@
 			if(istype(head_item, /obj/item/reagent_containers/pill))
 				for(var/datum/action/item_action/hands_free/activate_pill/pill_action in head_item.actions)
 					qdel(pill_action)
+			if(istype(head_item, /obj/item/organ))
+				var/obj/item/organ/organ = head_item
+				if(organ.organ_flags & ORGAN_UNREMOVABLE)
+					continue
```

Comment:
This whole `for(var/obj/item/head_item in src)` loop seems really branchy and weird...

Reply:
Agreed, this just needs a rework, but I wont be able to do any coding for a while